### DISCUSSION TITLE
xfce terminal in 13

### DISCUSSION THREAD

**Discussion by ghost** [i3/i3#6123] [Q&A] [answered]
I was wondering how to use the xfce terminal in i3. The only reference to using xfce with i3 was the xfce4-panel. I want to have the full i3 with just the xfce terminal, but can't figure out how to add it to the config.

**Comment by orestisfl** [comment, marked as answer]
You just need to launch the xfce terminal, there's usually nothing special about which terminal you are using. You can either bind the a key you want in your config, see https://i3wm.org/docs/userguide.html#exec or `export TERMINAL=xfce4-terminal` before launching i3.

**Reply by ghost** [reply]
Thanks for the help. I’m somewhat new to tilling window managers.
